### PR TITLE
[WIP] New logged in dropdown interface

### PR DESCRIPTION
**Pull request by dzucconi**
No merge yet.

TODO

- [x] Wire up channel create
- [x] Wire up notifications
- [x] Mobile

Currently looks like:

dropdown2

### CODE REVIEW

File: apps/block/templates/_sidebar.jade [hunk 1]
```diff
@@ -6,7 +6,7 @@
       include info/_description
       include info/_connections
 
-      if user && user.has('name')
+      if user && user.has('username')
```
Comment (dzucconi) [inline apps/block/templates/_sidebar.jade]: @broskoski I noticed this check for has username in a few places. Is it actually possible for there to be a user being rendered but them to not have a username?

Comment (broskoski) [inline apps/block/templates/_sidebar.jade]: Yeah, the `arena-passport` middleware injects a user into `res.locals` that is just an empty backbone model

File: components/layout/client.coffee [hunk 2]
```diff
@@ -82,19 +83,21 @@ setupViews = ->
       offset: 3
 
   if mediator.shared.current_user.id
-    new UserMenuView
-      el: $('.dropdown--menu--user')
-    new NewChannelView
-      el: $('.dropdown--menu--new-channel')
-    new NotificationsView
-      el: $('.dropdown--menu--notifications')
+    initLoggedInNavigation $('.js-logged-in-navigation')
```
Comment (broskoski) [inline components/layout/client.coffee]: 👍

File: components/layout/variables.styl [hunk 3]
```diff
@@ -161,3 +161,16 @@ candy-striped(color)
     transparent 75%,
     transparent
   )
+
+outline-border(size, color, radius)
+  &:after
+    display block
```
Comment (broskoski) [inline components/layout/variables.styl]: 👍

File: components/logged_in_navigation/components/notifications/templates/notification.jade [hunk 4]
```diff
@@ -0,0 +1,37 @@
+- klasses = [];
+
+- group.isNew() ? klasses.push('Notification--unread') : '';
+- group.is_comment() ? klasses.push('Notification--comment') : '';
```
Comment (dzucconi) [inline components/logged_in_navigation/components/notifications/templates/notification.jade]: Do comments typically look different? I'm not doing anything with this class

File: components/logged_in_navigation/stylesheets/index.styl [hunk 5]
```diff
@@ -0,0 +1,7 @@
+.LoggedInNavigation
+  display flex
+
+@require '../components/channel_create/index'
+@require '../components/notifications/index'
+@require '../components/view_mode/index'
+@require '../components/you/index'
```
Comment (broskoski) [inline components/logged_in_navigation/stylesheets/index.styl]: So orderly!

File: components/logged_in_navigation/components/notifications/templates/notification.jade [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    = group.grouped_subject()
+
+  if group.connector()
+    | &nbsp;#{group.connector()}&nbsp;
+
+    a( class="Color--channel-#{group.target_privacy()}", href= group.single_target_link() )
+      = group.single_target()
+
+  .Notification__timestamp
+    = group.timestamp()
```
Comment (dzucconi) [inline components/logged_in_navigation/components/notifications/templates/notification.jade]: This template is largely the existing small_feed. Just spruced up a bit. I suspect these conditions can be cleaned up somehow but I haven't thought to hard about it.

File: components/logged_in_navigation/components/notifications/templates/notification.jade (hunk 4 above)
Comment (broskoski) [inline components/logged_in_navigation/components/notifications/templates/notification.jade]: Mm it's got a bit of a different format, but it should look more or less the same:
screenshot 2017-05-26 17 30 58

Comment (broskoski) [inline components/logged_in_navigation/components/notifications/templates/notification.jade]: Lol "lamar" @damassi

File: components/logged_in_navigation/components/notifications/templates/notification.jade (hunk 6 above)
Comment (broskoski) [inline components/logged_in_navigation/components/notifications/templates/notification.jade]: Yeah, I think we should leave it alone until there is a nicer GraphQL endpoint that can make this nicer to deal with.

File: components/logged_in_navigation/components/notifications/templates/notification.jade (hunk 4 above)
Comment (damassi) [inline components/logged_in_navigation/components/notifications/templates/notification.jade]: screen shot 2017-05-26 at 4 08 38 pm

Comment (broskoski) [inline components/logged_in_navigation/components/notifications/templates/notification.jade]: @damassi Lavar!

Comment (damassi) [inline components/logged_in_navigation/components/notifications/templates/notification.jade]: hahahaha

**Comment by dzucconi** [comment]
And mobile:

mobile-dropdown

File: components/logged_in_navigation/components/channel_create/index.jade [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+    .ChannelCreatePrivacy__selector
+      a.js-status(
+        href='#hm'
+        data-value='public'
+        data-selected=(channel && channel.get('status') === 'public' ? true : undefined)
+      ) Open
+      span.Separator
+      a.js-status(
+        href='#hm'
```
Comment (dzucconi) [inline components/logged_in_navigation/components/channel_create/index.jade]: Oh lol

**Pull request merged**